components/Table.js

```diff
@@ -42,6 +42,15 @@ export default function Table({
             return rowValue !== undefined ? filterValues.includes(String(rowValue)) : true;
           });
         },
+        matchCategory: (rows, id, filterValues) => {
```

Comment:
Added a new filter type since `matchEnum` checked from an array to a single value, whereas here we can have multiple filters but also the categories column is an array so have to compare it slightly different